### ISSUE TITLE
[Design] On-site Notifications: Finalize design

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#32293]
## Background
After completing our design intent we need to finalize design details for our MVP

## Tasks
- [ ] Follow up with Design and a11y stakeholders on design feedback
- [ ] Finalize [design ](https://app.mural.co/t/vsa8243/m/vsa8243/1640028268317/2ba4e1b2846c87f5d0bfc6bcb643bef84720bea1?sender=30b328dd-61fa-4e9c-a615-80a4db99566d)details for MVP 
- [ ] Transfer documentation to GH

## AC
- [ ] Finalized MVP designs and FE documentation exist in GH

**Comment by andaleliz** [comment]
Hey all, just trying to keep everything straight - are there existing tickets that came out of design intent to reference for this?

**Comment by hdjustice** [comment]
@andaleliz @tressaellen I think this ticket is duplicative.  We have the Update with MVP #33266, which covers this as well.
Also, in response to Liz' questions above, there was only one Design Intent ticket from last review for creating 'sections' of notifications (like in the Notification Center we originally designed for) #30607.  This would no longer be required at this point.  
The ticket for revisiting with Design Intent (since our scope changed) is #33301.  

In short, I would like to
- close this ticket as being duplicative of #33266
- close ticket #30607 as not being applicable any longer due to scope change
- keep #33301 open so we can inform Design Intent of our MVP update 
     - with the knowledge that we are submitting for experimental design on it, along with Payment Information effort
